src/python/pants/backend/python/target_types.py

```diff
@@ -254,10 +255,39 @@ def value_or_global_default(self, pex_binary_defaults: PexBinaryDefaults) -> boo
         return self.value
 
 
+class PexExecutionMode(Enum):
+    ZIPAPP = "zipapp"
+    UNZIP = "unzip"
+    VENV = "venv"
+
+
+class PexExecutionModeField(StringField):
+    alias = "execution_mode"
+    valid_choices = PexExecutionMode
+    expected_type = str
+    help = (
+        "The mode the generated PEX file will run in. The traditional PEX file runs in "
+        f"{PexExecutionMode.ZIPAPP.value!r} mode (See: https://www.python.org/dev/peps/pep-0441/). "
+        f"In general, faster cold start times can be attained using the "
+        f"{PexExecutionMode.UNZIP.value!r} mode which also has the benefit of allowing standard "
+        "use of `__file__` and filesystem APIs to access code and resources in the PEX. The "
```

Comment:
Would be good to add `\n\n` between these two paragraphs. Makes it easier to see that all the following sentences are about VENV mode.

Reply:
Fixed.